tensorflow/python/ops/control_flow_ops_test.py

```diff
@@ -22,6 +22,7 @@
 import numpy as np
 
 from tensorflow.python import tf2
+from tensorflow.contrib import summary
```

Comment:
TF core tests are not allowed to depend on contrib

Reply:
Ok, but this is fix for that problem, as mentioned in issue #24815 . Do You have any advice on how to rewrite test?

Reply:
Import the internal module, which I think is in tensorflow.python.ops.summary_v2